model/task/task.go

```diff
@@ -106,6 +106,12 @@ type Task struct {
 	NumDependents            int          `bson:"num_dependents,omitempty" json:"num_dependents,omitempty"`
 	OverrideDependencies     bool         `bson:"override_dependencies,omitempty" json:"override_dependencies,omitempty"`
 
+	// DistroAliases refer to the optional secondary distros that can be
+	// associated with a task. This is used for running tasks in case there
+	// is excess capcacity in a distro. Despite the variable name, this is a
```

Comment:
"excess load". Generally, "capacity" should be used to describe ability to absorb load, in our case the room we have for tasks, e.g. lots of hosts is "high capacity", too few is "low capacity".

Reply:
Thanks for the change. I misunderstood the functionality, but it is clear now.